app/src/main/java/ru/flicksbox/user/data/LoginBody.kt

```diff
@@ -0,0 +1,8 @@
+package ru.flicksbox.user.data
+
+import com.google.gson.annotations.SerializedName
+
+data class LoginBody(
```

Comment:
I suggest to rename to LoginDTO, or LoginRequestDTO, to conform with DTO naming approach